packages/circuit-ui/components/Modal/Modal.module.css

```diff
@@ -18,111 +35,75 @@
   );
 }
 
-@media (max-width: 479px) {
-  .base {
-    right: 0;
-    bottom: 0;
-    left: 0;
-    transition: transform var(--cui-transitions-default);
-    transform: translateY(100%);
-  }
-
-  .base::after {
-    height: var(--cui-spacings-mega);
-  }
-}
-
 @media (min-width: 480px) {
-  .base {
-    top: 50%;
-    left: 50%;
-    border-radius: var(--cui-border-radius-mega);
-    opacity: 0;
-    transition: opacity var(--cui-transitions-slow);
-    transform: translate(-50%, -50%);
-  }
-
-  .base::after {
+  .dialog::after {
     height: var(--cui-spacings-giga);
     border-bottom-right-radius: var(--cui-border-radius-mega);
     border-bottom-left-radius: var(--cui-border-radius-mega);
   }
 }
 
-/* Variants */
-
 @media (max-width: 479px) {
-  .contextual {
```

Comment:
RIP? Why?

Reply:
Contextual styles are the default styles. Ideally I would have change the variant prop to a boolean `isImmersive` but kept it for backwards compatibility's sake. Do you think its better to distinguish the two ?